src/notebooks/controllers/vscodeNotebookController.node.ts

```diff
@@ -486,12 +485,19 @@ export class VSCodeNotebookController implements Disposable {
         // Execution should be ended elsewhere
     }
 
-    @chainable()
```

Comment:
This wasn't required, we don't need to chain starting the kernel.

Reply:
You should remove the comment below then.